_clients/java-rest-high-level.md

```diff
@@ -25,7 +22,33 @@ To start using the OpenSearch Java high-level REST client, ensure that you have
 
 You can now start your OpenSearch cluster. The OpenSearch 1.x high-level REST client works with the 1.x versions of OpenSearch.
 
-The following example uses credentials that come with the default OpenSearch configuration. If you’re using the high-level REST client with your own OpenSearch cluster, be sure to change the code to use your own credentials.
+## Security
+
+This code example uses basic credentials that come with the default OpenSearch configuration. If you’re using the OpenSearch Java high-level REST client with your own OpenSearch cluster, be sure to change the code to use your own credentials.
+{: .note}
+
+Before you can securely connect to an OpenSearch cluster, you must first add your root certificates to a truststore. If you don't already have a custom truststore, the following command creates a truststore and adds in a certificate.
```

Comment:
See the technically corrected version below. Feel free to correct/rephrase to suit better.

```suggestion
Before using the REST client in your Java application, you need to configure the application truststore. If you are using the self-signed certificates or demo configurations, you can create a custom truststore and add the root authority certificates to the truststore using `Java keytool` as shown below... 
.
.
Note - You don't need to configure the truststore if you have setup OpenSearch with certificates from a trusted Certificate Authority (CA). 
```